test/e2e-ansible/e2e_ansible_olm_test.go

```diff
@@ -26,17 +26,13 @@ var _ = Describe("Integrating ansible Projects with OLM", func() {
 		const operatorVersion = "0.0.1"
 
 		It("should generate and run a valid OLM bundle and packagemanifests", func() {
-			By("building the bundle")
-			err := tc.Make("bundle", "IMG="+tc.ImageName)
-			Expect(err).NotTo(HaveOccurred())
-
 			By("building the operator bundle image")
-			err = tc.Make("bundle-build", "BUNDLE_IMG="+tc.BundleImageName)
+			err := tc.Make("bundle-build", "BUNDLE_IMG="+tc.BundleImageName)
 			Expect(err).NotTo(HaveOccurred())
 
 			if tc.IsRunningOnKind() {
 				By("loading the bundle image into Kind cluster")
```

Comment:
You don't need to load the bundle image since `run bundle` would be able to pull it itself.
```suggestion
			if tc.IsRunningOnKind() {
				By("loading the bundle image into Kind cluster")
```